### PR TITLE
[TW-1551] Create overview for /docs/getting-started/basics

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
Add overview topic for Get started > Postman basics

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM! Excellent use of the word "launch". 😀

File: src/pages/docs/getting-started/basics/postman-basics.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Postman has a variety of tools you can use to work with APIs and collaborate with your team. [Learn how to navigate in Postman](/docs/getting-started/basics/navigating-postman/) so you can always find the features you're looking for.
+
+Need more help? Postbot is your AI assistant for API workflows. [You can ask Postbot](/docs/getting-started/basics/about-postbot/) to add tests to requests, visualize responses, write documentation, and more.
+
+## Send API requests from Postman
+
+[The Postman Agent](/docs/getting-started/basics/about-postman-agent/#selecting-a-postman-agent-for-requests) enables you to send API requests from a browser using the Postman web app. You can manually select an agent, or Postman can select the best agent for you.
+
+If you're not logged into a Postman account, you can still send multi-protocol requests using the [lightweight API client](/docs/getting-started/basics/using-api-client/).
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/basics/postman-basics.md:48]: Thanks for catching this!

[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/basics/postman-basics.md]: Style guide changes mention to use "sign in" and "in to".

File: src/pages/docs/getting-started/basics/postman-basics.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Need more help? Postbot is your AI assistant for API workflows. [You can ask Postbot](/docs/getting-started/basics/about-postbot/) to add tests to requests, visualize responses, write documentation, and more.
+
+## Send API requests from Postman
+
+[The Postman Agent](/docs/getting-started/basics/about-postman-agent/#selecting-a-postman-agent-for-requests) enables you to send API requests from a browser using the Postman web app. You can manually select an agent, or Postman can select the best agent for you.
+
+If you're not signed in to a Postman account, you can still send multi-protocol requests using the [lightweight API client](/docs/getting-started/basics/using-api-client/).
+
+> For the best experience sending API requests, [download the Postman desktop app](/docs/getting-started/first-steps/get-postman/) and [sign into your Postman account](/docs/getting-started/first-steps/sign-in-to-postman/).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/getting-started/basics/postman-basics.md:50]:
```suggestion
> For the best experience sending API requests, [download the Postman desktop app](/docs/getting-started/first-steps/get-postman/) and [sign in to your Postman account](/docs/getting-started/first-steps/sign-in-to-postman/).
```

File: src/pages/docs/getting-started/basics/postman-basics.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Save and share your work in Postman
+
+When you're signed in to your Postman account, all of your work is [automatically synced to the Postman cloud](/docs/getting-started/basics/syncing/). That means you can access all of your collections, requests, and other data anywhere you're signed into Postman.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/getting-started/basics/postman-basics.md:54]:
```suggestion
When you're signed in to your Postman account, all of your work is [automatically synced to the Postman cloud](/docs/getting-started/basics/syncing/). That means you can access all of your collections, requests, and other data anywhere you're signed in to Postman.
```

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks great!

File: src/pages/docs/getting-started/basics/postman-basics.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/basics/postman-basics.md:50]: Thanks for noticing all these!

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Pull request merged**